src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Extras/RichTextEditor/BitRichTextEditorDemo.razor.scss

```
@@ -3,95 +3,13 @@
 }
 
-.position-btn {
-    gap: rem2(16px);
-    display: flex;
-    flex-flow: column nowrap;
-
-    div {
-        display: flex;
-        justify-content: space-between;
-    }
+::deep .custom-rte-toolbar {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)